packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -533,6 +533,28 @@ class _CupertinoTextFieldState extends State<CupertinoTextField> with AutomaticK
     _editableTextKey.currentState.showToolbar();
   }
 
+  void _handleDragSelectionStart(DragStartDetails details) {
```

Comment:
This is mouse specific right? Maybe just have the specific mouse word in here since this class is where the rubber concretely meets the road.